source/renderer/app/containers/settings/categories/SupportSettingsPage.js

```diff
@@ -1,46 +1,89 @@
 import { observer, inject } from 'mobx-react';
+import { defineMessages, intlShape } from 'react-intl';
 import SupportSettings from '../../../components/settings/categories/SupportSettings';
 import type { InjectedProps } from '../../../types/injectedPropsType';
-import BugReportDialog from '../../../components/profile/bug-report/BugReportDialog';
 import { generateFileNameWithTimestamp } from '../../../../../common/utils/files';
-import { openExternalUrlChannel } from '../../../ipc/open-external-url';
+import { getSupportUrl } from '../../../utils/network';
+import successIcon from '../../../assets/images/success-small.inline.svg';
+import NotificationMessage from '../../../components/widgets/NotificationMessage';
+import { DOWNLOAD_LOGS_SUCCESS_DURATION } from '../../../config/timingConfig';
+
+const messages = defineMessages({
+  supportRequestLinkUrl: {
+    id: 'settings.support.reportProblem.linkUrl',
+    defaultMessage: '!!!https://iohk.zendesk.com/hc/en-us/categories/360000877653-Daedalus-wallet-mainnet',
+    description: '"Support request" link URL in the "Report a problem" section on the support settings page.',
+  },
+  downloadLogsSuccess: {
+    id: 'settings.support.reportProblem.downloadLogsSuccessMessage',
+    defaultMessage: '!!!Logs were downloaded',
+    description: 'Success message for download logs.',
+  },
+});
 
 @inject('stores', 'actions') @observer
 export default class SupportSettingsPage extends Component<InjectedProps> {
 
-  static defaultProps = { actions: null, stores: null };
-
-  handleExternalLinkClick = (event: MouseEvent, url: string) => {
-    event.preventDefault();
-    openExternalUrlChannel.send(url);
+  static contextTypes = {
+    intl: intlShape.isRequired,
   };
 
-  handleSupportRequestClick = () => {
-    this.props.actions.dialogs.open.trigger({
-      dialog: BugReportDialog,
-    });
+  static defaultProps = { actions: null, stores: null };
+
+  handleSupportRequestClick = async (event: SyntheticEvent<HTMLButtonElement>) => {
+    event.persist();
+    const { intl } = this.context;
+    const supportRequestLinkUrl = intl.formatMessage(messages.supportRequestLinkUrl);
+    const locale = this.props.stores.profile.currentLocale;
+    const supportUrl = await getSupportUrl(supportRequestLinkUrl, locale);
+    this.props.stores.app.openExternalLink(supportUrl);
   };
 
   handleDownloadLogs = () => {
     // TODO: refactor this direct access to the dialog api
     const fileName = generateFileNameWithTimestamp();
+    const { actions } = this.props;
     const destination = global.dialog.showSaveDialog({
       defaultPath: fileName,
     });
     if (destination) {
-      this.props.actions.profile.downloadLogs.trigger({ fileName, destination, fresh: true });
+      actions.profile.downloadLogs.trigger({ fileName, destination, fresh: true });
     }
+    const { id, duration } = this.notification;
+    actions.notifications.open.trigger({ id, duration, });
```

Comment:
You forgot to check if the downloaded logs are *actually* successfully saved before showing the notification!